.github/workflows/ci.yml

```diff
@@ -17,6 +17,10 @@ on:
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
 
+env:
+  # Run the step only on PRs from the main repo, a scheduled, or triggered run.
+  RUN_ON_FORK_PR_AND_SCHEDULE_AND_TRIGGER: github.event.pull_request.head.repo.full_name == github.repository || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
```

Comment:
Maybe we can consider changing the name of this variable? If it is not running on fork PRs then we can remove the word FORK.

Reply:
shoot you're right.

Reply:
PR has been updated